code/css-assignment-3/style.css

```diff
@@ -0,0 +1,157 @@
+body {
+    display: flex;
+    margin: 0;
+}
+
+#profile {
+    background-color: #f9b8b6;
+    padding: 1em;
+    margin: 1em;
+}
+
+#profile > img {
+    outline: 0.3rem solid #fff;
+    margin: 0.5em 0;
+}
+
+#profile > span {
+    font-size: 1.3em;
+    font-weight: 600;
+    padding-bottom: 1em;
+}
+
+.card {
+    border: 3px solid #006d9e;
+    margin: 0.2rem;
+}
+
+img {
+    width: 100%;
+}
+
+header {
+    background-color: #a7cddf;
+    font-size: 2em;
+    display: flex;
+    justify-content: center;
+}
+
+@media screen and (min-width: 500px) {
+    body{
+       display: flex;
+       flex-direction: column;
+       }
+    #profile { 
+    img {
+        width: 85vw;
+     }   
+    }
+
+    #gallery {
+        .card.right {
+            display: flex;
+            flex-direction: row-reverse;
+        }
+        .card {
+            display: flex;
+            justify-content: flex-end;
+                       }
+        img {
+            width: 70vw;
+        }
+        span {
+            padding: 20px;
+            align-items: flex-start;
+        }
+
+    }
+
+    }
+ 
+    @media screen and (min-width: 750px) {
+        #gallery {
+          display: flex;
+           gap: 15px;
+           flex-direction: row;
+           justify-content: center;
+           align-items: center;
+           flex-wrap: wrap;
+
+        .card.right {
+            width: 25vw;
+            flex-direction: row;
+            flex-direction: column;
+            flex-grow: 1;
+
+         img {
+            
+            width: 100%;
+    
+        }
+    
+        }
+
+        .card {
+            width: 25vw;
+             flex-direction: column;
+             flex-grow: 1;
+            img {
+                
+                width: 100%;
+        
+
+            }
+        }
+        }
+
+        }
+            
+ @media screen and (min-width: 1000px) {
+    #page-wrapper {
+       display: grid;
+       flex-grow: 1;
+       grid-template-columns: 1fr  2fr;
+       grid-template-areas: 
+       "header header"
+       "profile main";  
+    }
+    header {
+        grid-area: header;
+      }
+    #profile {
+        grid-area: profile;
+        width: 20vw;
+        img {
+            width: 20vw;
+        }
+      }
+    #gallery {
+        grid-area: main;
+        flex-wrap: nowrap;
```

Comment:
This was an interesting choice. Why nowrap? In the example, the images definitely wrap!